### DISCUSSION TITLE
Add `useCookie`

### DISCUSSION THREAD

**Discussion by antoinekm** [juliencrn/usehooks-ts#409] [Ideas]
Adding a use cookie hook similar to the `useLocalStorage` could be funny

**Comment by gabriellopes00** [comment]
I've needed this feature in one of my projects, and I implemented a hook exactly like `useLocalStorage` and `useSessionStorage` but instead of the storages, it saves the data into the cookies, in addition to the key name and the default value, the hooks receive the expiration time of the cookie.... To access cookies in react I used [js-cookie](https://github.com/js-cookie/js-cookie). I can open a PR with the code, what do you guys think about it? @juliencrn

**Reply by LonelyFellas** [reply]
#574

**Reply by Strajk** [reply]
Correct PR https://github.com/juliencrn/usehooks-ts/pull/576
